app/lib/searchValidator.js

```diff
@@ -4,14 +4,17 @@ function isEmpty(search) {
   return !search || !search.trim();
 }
 
-function validateSearch(search) {
+function validateSearch(searchTermName, searchTermPostcode) {
```

Comment:
Perhaps a better name for this would be checkForEmptySearch. I realise that it hasn't been renamed in this work.